### DISCUSSION TITLE
I deleted the app yet the background still won't get removed

### DISCUSSION THREAD

**Discussion by UniversalEnd** [rocksdanister/lively#607] [Q&A] [answered]
I don't understand how git.hub works or the lively wallpapers app. I forgot to remove the background before deleting the application. I tried to re-install the app but it won't download since there's another lively wallpapers app open. I tried restarting my laptop yet the background still wouldn't go away. I tried looking for something online and found this website. I can't figure out how to remove the background. I know this is my mistake but if you could help me fix it I would be really happy. (windows 10)

**Comment by rocksdanister** [comment, marked as answer]
1. From where did you download Lively, which version number?
2. How exactly did you uninstall?

There is code in the uninstaller to kill running Lively and mutex check also ensure uninstaller only proceed after termination:
https://github.com/rocksdanister/lively/blob/dev-v1.0-fluent-netcore/src/installer/Script.iss

It is impossible for this to happen unless somehow you downloaded year old version of Lively without these checks or did not run the uninstaller.

**Reply by UniversalEnd** [reply]
1. I downloaded v1.7.0.2 from https://rocksdanister.github.io/lively/
2. I uninstalled it by dragging the lively app into the recycling bin and deleting it

**Reply by rocksdanister** [reply]
>I uninstalled it by dragging the lively app into the recycling bin and deleting it

You deleted shortcut.. that is not "Uninstall"

The initial setup wizard also showed how to quit running Lively:
![tutorial_systray](https://user-images.githubusercontent.com/17554161/125209850-7550f880-e2b9-11eb-9e1c-c55457614fa6.png)

Follow these instructions for uninstall:
https://in.pcmag.com/migrated-65955-windows-10/141107/how-to-uninstall-programs-in-windows-10

**Reply by UniversalEnd** [reply]
Thank you very much for the help, it worked now
